postgis/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG PG_VERSION_TAG
-FROM timescale/timescaledb:1.7.1-${PG_VERSION_TAG}
+FROM timescale/timescaledb:2.0.0-beta5-${PG_VERSION_TAG}
```

Comment:
shouldnt this be timescaledev since we push beta images to dev

Reply:
@svenklemm It was explicitly decided to push them into normal repo (not dev) on a standup meeting last Thursday.

Reply:
ok if its intentional thats fine with me